hbase-oss/pom.xml

```diff
@@ -311,6 +318,7 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
       <version>${hadoop.version}</version>
+      <type>test-jar</type>
       <scope>test</scope>
     </dependency>
     <dependency>
```

Comment:
Adding these allowed me to get build issues [reported here](https://jira.apache.org/jira/browse/HBASE-22574?focusedCommentId=16862654&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16862654) sorted.